package.json

```diff
@@ -34,12 +36,15 @@
     "gulp-sass": "^2.1.1",
     "gulp-sourcemaps": "^1.6.0",
     "gulp-watch": "^4.3.5",
+    "jest-cli": "^0.8.2",
     "marked": "^0.3.5",
     "minify": "^2.0.2",
     "mocha": "^2.3.4",
+    "react-addons-test-utils": "^0.14.3",
     "webpack-stream": "^3.1.0"
   },
   "dependencies": {
+    "body-parser": "^1.14.2",
```

Comment:
Some packages got loaded into dependencies instead of devDependencies and vice-versa. Fixed all the ones I knew for sure.